components/Hooks/useLocalStorage.js

```diff
@@ -0,0 +1,45 @@
+import { useCallback, useEffect, useState } from "react";
```

Comment:
if this hook suits your purpose - keep on going with this.
Anyway - we found some disadvantages and suggest to use the [`local Sorage-state`](https://www.npmjs.com/package/use-local-storage-state). 
Keep in mind and switch only if you reach the limits of this hook ;)